core/tools/generate_names.cc

```diff
@@ -59,6 +59,7 @@ NameDef names[] = {
 
     // Used in parser for error recovery
     {"methodNameMissing", "<method-name-missing>"},
+    {"methodDefNameMissing", "<method-def-name-missing>"},
```

Comment:
Do we need to add this to all the places that already check `methodNameMissing`?

Reply:
Good question: the only two places we use methodNameMissing to silence errors are:

- call to this method, as an error was already reported. it's not possible at the moment for this NameRef to show up in a `Send` node as far as I can see, so not there.
- in a completion result. we do not yet have an `isMethodDef` case in completion.cc yet, though we will when I get around to polishing #5886.